go.mod

```diff
@@ -2,18 +2,21 @@ module github.com/KyberNetwork/promm-sdk-go
 
 go 1.17
 
+replace github.com/daoleno/uniswap-sdk-core v0.1.5 => github.com/KyberNetwork/uniswap-sdk-core v0.1.5
```

Comment:
should be removed?

Reply:
no, this is the intention that Truc and I had discussed

We fork the code so that even if `daoleno` decides to remove it in the future, we still have the code. And we can customize whatever we want

But we want to use `import "github.com/daoleno/uniswap-sdk-core"` inside our code base so that whenever there is a change from upstream (daoleno/uniswap-sdk-core or daoleno/uniswapv3-sdk), we can easily cherry-pick it

Similarly in kyberswap-aggregator in the future when we need to `import "github.com/daoleno/uniswapv3-sdk"`, we can use Golang `replace` so that kyberswap-aggregator actually uses `github.com/KyberNetwork/uniswapv3-sdk` (though the import statement still refers to `github.com/daoleno/uniswapv3-sdk`)

Reply:
@trucdoankyber :

```
replace (
  github.com/daoleno/uniswap-sdk-core v0.1.5 => github.com/KyberNetwork/uniswap-sdk-core v0.1.5
)
```
looks a bit better